themes/src/main/resources/theme/keycloak/login/theme.properties

```diff
@@ -2,8 +2,7 @@ parent=base
 import=common/keycloak
 
 styles=css/login.css
-stylesCommon=node_modules/patternfly/dist/css/patternfly.min.css node_modules/patternfly/dist/css/patternfly-additions.min.css lib/zocial/zocial.css
-
+stylesCommon=node_modules/patternfly/dist/css/patternfly.min.css web_modules/@patternfly/react-core/dist/styles/app.css web_modules/@patternfly/react-core/dist/styles/base.css node_modules/patternfly/dist/css/patternfly-additions.min.css lib/zocial/zocial.css
```

Comment:
@Pepo48 Could you please remove the zocial css? There is no need for that anymore. It may avoid some merge conflicts.

Reply:
@mabartos thanks for the review! Both issues should be fixed now.